Formula/allure.rb

```diff
@@ -1,8 +1,7 @@
 class Allure < Formula
   desc "Flexible lightweight test report tool"
   homepage "https://github.com/allure-framework/allure2"
-  url "https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.7.0/allure-2.7.0.zip"
-  sha256 "9408ff5c11abda7491431539b38a724689b2172a1fdd1b1dacca5f4ab77a402b"
```

Comment:
The `sha256` should be removed.

Reply:
There are no `sha256` provided on GitHub releases page, and it differs from the bintray one

Reply:
Should I provide the one calculated locally?

Reply:
Yes, but `allure` upstream will need to confirm the `sha256` as the GitHub zip differs from the Bintray zip.

Reply:
@eroshenkoam could you take a look here? ⬆️

Reply:
I'm talking with bintray support right now. I need more time to make this decision.

Reply:
Old URL is back online, so I guess I can close this PR